nss-ice-phantoms/WorkforceManagement
# Rr employees

# Description

Issue #4 complete.

## Type of change

# Testing Instructions

1. `git fetch --all`
2. `git checkout rr-employees`
3. Run BangazonWorkforce
4. Use the attached file to add additional records to your DB for testing
5. Verify that Edit link by any employee in the list takes you to the edit page and displays First Name, Last Name, Full Name, Department Name, a list of Computers (the employee's currently assigned computer should be marked with ***) and two lists of Training Programs, those the employee is enrolled in and those they are not enrolled in.
6. Verify that changing employee data updates database.  Changing computer should create a new record in ComptuerEmployee table and also update the "UnassignDate" field in the old record.  Selecting Training Programs in the "Enrolled" list will unenroll the employee, you should see deletion of the associated record in EmployeeTraining.  Selecting Training Programs in the "Unenrolled" list will enroll the employee, you should see addition of a record in EmployeeTraining.  

# Checklist:

[Data2.txt](https://github.com/nss-ice-phantoms/WorkforceManagement/files/3074642/Data2.txt)
